config/initializers/sidekiq.rb

```diff
@@ -1,10 +1,10 @@
 Sidekiq.configure_server do |config|
-  config.logger = Rails.logger
-  config.redis = { url: "redis://127.0.0.1:6380/5" } if Rails.env.development?
+  config.logger = Rails.logger unless Rails.env.development?
+  config.redis = { url: ENV.fetch("REDIS_URL", "redis://localhost:6379/1") }
```

Comment:
consider removing as REDIS_URL Is default 
see using [an env var](https://github.com/sidekiq/sidekiq/wiki/Using-Redis#using-an-env-variable)